tests/testthat/test_Backend.R

```diff
@@ -0,0 +1,5 @@
+test_that(".valid.Backend.processingQueue works", {
```

Comment:
Sorry that I forgot to commit and push my `test_Backend.R` yesterday.

Reply:
👍